src/main/io/gps.h

```diff
@@ -145,9 +329,19 @@ typedef struct gpsData_s {
 
     uint8_t ackWaitingMsgId;        // Message id when waiting for ACK
     uint8_t ackTimeoutCounter;      // Ack timeout counter
-    ubloxAckState_e ackState;
-    bool ubloxUsePVT;
-    bool ubloxUseSAT;
+    ubloxAckState_e ackState;       // Ack State
+    uint8_t updateRate;
+    bool ubloxM7orAbove;
+    bool ubloxM8orAbove;
+    bool ubloxM9orAbove;
```

Comment:
Use `ubloxVersion_e ubloxVersion` instead
```suggestion
```

Reply:
Maybe when UBlox make their next incompatible version we can refactor this, but for now can we leave it as is please.